t/node/remote-addr-ipv6.t

```diff
@@ -114,6 +114,7 @@ request sent: 59
 received: HTTP/1.1 200 OK
 received: Content-Type: text/plain
 received: Connection: close
+received: Server: APISIX/1.4.1
```

Comment:
the version is changing always.
We need to use regular matching.

Reply:
Yeah agreed. We do not want to update TCs every time we do a version bump up.
Wanted to check if the pipeline is a success or not.